src/data/nat/basic.lean

```diff
@@ -741,6 +741,17 @@ begin
   { intros h hP, exact hP }
 end
 
+/-- A subset of ℕ containing b:ℕ and closed under succ contains n ≥ b -/
+lemma ind_set' (b : ℕ) (S : set ℕ) (hb : b ∈ S) (h_ind: ∀ k : ℕ, k ∈ S → k.succ ∈ S) :
```

Comment:
```suggestion
lemma ind_set' {b : ℕ} {S : set ℕ} (hb : b ∈ S) (h_ind: ∀ k : ℕ, k ∈ S → k + 1 ∈ S) :
```
You can also move `n` and `hn : b <= n` left of the `:`, and make `n` an implicit argument.